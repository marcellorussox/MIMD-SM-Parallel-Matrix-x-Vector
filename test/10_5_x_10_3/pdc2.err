<============== TEST 100000 x 100 ==============>


<====== NUMBER OF THREADS: 1 ======>

<======= TEST 1 =======>

Overall time: 0.023870

<======================>


<======= TEST 2 =======>

Overall time: 0.023475

<======================>


<======= TEST 3 =======>

Overall time: 0.021376

<======================>


<======= TEST 4 =======>

Overall time: 0.028540

<======================>


<======= TEST 5 =======>

Overall time: 0.025830

<======================>


<====== NUMBER OF THREADS: 2 ======>

<======= TEST 1 =======>

Overall time: 0.022894

<======================>


<======= TEST 2 =======>

Overall time: 0.023613

<======================>


<======= TEST 3 =======>

Overall time: 0.021770

<======================>


<======= TEST 4 =======>

Overall time: 0.024691

<======================>


<======= TEST 5 =======>

Overall time: 0.025642

<======================>


<====== NUMBER OF THREADS: 3 ======>

<======= TEST 1 =======>

Overall time: 0.024422

<======================>


<======= TEST 2 =======>

Overall time: 0.024815

<======================>


<======= TEST 3 =======>

Overall time: 0.023265

<======================>


<======= TEST 4 =======>

Overall time: 0.023868

<======================>


<======= TEST 5 =======>

Overall time: 0.024707

<======================>


<====== NUMBER OF THREADS: 4 ======>

<======= TEST 1 =======>

Overall time: 0.025972

<======================>


<======= TEST 2 =======>

Overall time: 0.023974

<======================>


<======= TEST 3 =======>

Overall time: 0.024437

<======================>


<======= TEST 4 =======>

Overall time: 0.023595

<======================>


<======= TEST 5 =======>

Overall time: 0.023930

<======================>


<====== NUMBER OF THREADS: 5 ======>

<======= TEST 1 =======>

Overall time: 0.026287

<======================>


<======= TEST 2 =======>

Overall time: 0.023928

<======================>


<======= TEST 3 =======>

Overall time: 0.024481

<======================>


<======= TEST 4 =======>

Overall time: 0.022423

<======================>


<======= TEST 5 =======>

Overall time: 0.024622

<======================>


<====== NUMBER OF THREADS: 6 ======>

<======= TEST 1 =======>

Overall time: 0.024026

<======================>


<======= TEST 2 =======>

Overall time: 0.023698

<======================>


<======= TEST 3 =======>

Overall time: 0.023578

<======================>


<======= TEST 4 =======>

Overall time: 0.023096

<======================>


<======= TEST 5 =======>

Overall time: 0.022253

<======================>


<====== NUMBER OF THREADS: 7 ======>

<======= TEST 1 =======>

Overall time: 0.022984

<======================>


<======= TEST 2 =======>

Overall time: 0.024296

<======================>


<======= TEST 3 =======>

Overall time: 0.023358

<======================>


<======= TEST 4 =======>

Overall time: 0.026551

<======================>


<======= TEST 5 =======>

Overall time: 0.023062

<======================>


<====== NUMBER OF THREADS: 8 ======>

<======= TEST 1 =======>

Overall time: 0.026323

<======================>


<======= TEST 2 =======>

Overall time: 0.024752

<======================>


<======= TEST 3 =======>

Overall time: 0.023847

<======================>


<======= TEST 4 =======>

Overall time: 0.023057

<======================>


<======= TEST 5 =======>

Overall time: 0.023145

<======================>
