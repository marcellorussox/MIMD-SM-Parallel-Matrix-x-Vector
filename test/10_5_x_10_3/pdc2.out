<============== TEST 100000 x 1000 ==============>


<====== NUMBER OF THREADS: 1 ======>

<======= TEST 1 =======>

Overall time: 1.147136

<======================>


<======= TEST 2 =======>

Overall time: 1.147148

<======================>


<======= TEST 3 =======>

Overall time: 1.150547

<======================>


<======= TEST 4 =======>

Overall time: 1.147587

<======================>


<======= TEST 5 =======>

Overall time: 1.150138

<======================>


<====== NUMBER OF THREADS: 2 ======>

<======= TEST 1 =======>

Overall time: 0.580966

<======================>


<======= TEST 2 =======>

Overall time: 0.581652

<======================>


<======= TEST 3 =======>

Overall time: 0.580641

<======================>


<======= TEST 4 =======>

Overall time: 0.584083

<======================>


<======= TEST 5 =======>

Overall time: 0.580634

<======================>


<====== NUMBER OF THREADS: 3 ======>

<======= TEST 1 =======>

Overall time: 0.393143

<======================>


<======= TEST 2 =======>

Overall time: 0.393621

<======================>


<======= TEST 3 =======>

Overall time: 0.394292

<======================>


<======= TEST 4 =======>

Overall time: 0.391820

<======================>


<======= TEST 5 =======>

Overall time: 0.392448

<======================>


<====== NUMBER OF THREADS: 4 ======>

<======= TEST 1 =======>

Overall time: 0.296284

<======================>


<======= TEST 2 =======>

Overall time: 0.299130

<======================>


<======= TEST 3 =======>

Overall time: 0.297903

<======================>


<======= TEST 4 =======>

Overall time: 0.299010

<======================>


<======= TEST 5 =======>

Overall time: 0.297193

<======================>


<====== NUMBER OF THREADS: 5 ======>

<======= TEST 1 =======>

Overall time: 0.240282

<======================>


<======= TEST 2 =======>

Overall time: 0.238242

<======================>


<======= TEST 3 =======>

Overall time: 0.237762

<======================>


<======= TEST 4 =======>

Overall time: 0.238850

<======================>


<======= TEST 5 =======>

Overall time: 0.240284

<======================>


<====== NUMBER OF THREADS: 6 ======>

<======= TEST 1 =======>

Overall time: 0.200546

<======================>


<======= TEST 2 =======>

Overall time: 0.202356

<======================>


<======= TEST 3 =======>

Overall time: 0.201922

<======================>


<======= TEST 4 =======>

Overall time: 0.202447

<======================>


<======= TEST 5 =======>

Overall time: 0.200544

<======================>


<====== NUMBER OF THREADS: 7 ======>

<======= TEST 1 =======>

Overall time: 0.174773

<======================>


<======= TEST 2 =======>

Overall time: 0.176378

<======================>


<======= TEST 3 =======>

Overall time: 0.177423

<======================>


<======= TEST 4 =======>

Overall time: 0.177197

<======================>


<======= TEST 5 =======>

Overall time: 0.176815

<======================>


<====== NUMBER OF THREADS: 8 ======>

<======= TEST 1 =======>

Overall time: 0.158039

<======================>


<======= TEST 2 =======>

Overall time: 0.157084

<======================>


<======= TEST 3 =======>

Overall time: 0.157582

<======================>


<======= TEST 4 =======>

Overall time: 0.157837

<======================>


<======= TEST 5 =======>

Overall time: 0.158856

<======================>
